phpcs.xml.dist

```diff
@@ -3,15 +3,17 @@
   <description>The coding standard.</description>
 
   <file>.</file>
+
   <exclude-pattern>vendor/</exclude-pattern>
 
   <!-- Show progress of the run -->
   <arg value="p"/>
 
   <arg name="extensions" value="php,module,inc,install,test,profile,theme,css,info,txt,yml"/>
-  <config name="drupal_core_version" value="8"/>
+  <config name="drupal_core_version" value="9"/>
 
   <rule ref="Drupal">
+    <!-- <exclude name="Drupal.Files.TxtFileLineLength.TooLong"/> -->
```

Comment:
Why is this comment added? It has no effect on the coding standards check (it's a comment).

Reply:
fixed